core/core_builders.py

```diff
@@ -228,42 +253,59 @@ def next_tag(self):
         for project_name, project in iteritems(projects):
             part_indexes[project_name] = part_index
             for part in project:
-                f.write("\t{ \"" + escape_string(part["License"][0]) + "\", "
-                        + "&COPYRIGHT_INFO_DATA[" + str(part["file_index"]) + "], "
-                        + "&COPYRIGHT_INFO_DATA[" + str(part["copyright_index"]) + "], "
-                        + str(len(part["Files"])) + ", "
-                        + str(len(part["Copyright"])) + " },\n")
+                f.write(
```

Comment:
Source string wasn't the prettiest, but this forced wrapping on each operator is really bad.